sirepo/auth_db.py

```diff
@@ -324,8 +325,16 @@ class UserRoleInvite(UserDbBase):
 
         @classmethod
         def get_moderation_request_rows(cls):
+            import sirepo.auth
             with sirepo.util.THREAD_LOCK:
-                return cls.search_all_by(status='pending') + cls.search_all_by(status='clarify')
+                t = sirepo.auth.get_module('email').UserModel
+                return [r[1].as_pkdict().pkupdate(
```

Comment:
Thanks for figuring out the query. sqlalchemy never makes much sense to me...

This is a bit of code golf but I see two minor issues with your query
1. You select everything from the auth_email_user_t when all you need is the user_name. We have such few users it doesn't really matter but if we had enough users to the point of being careful about what we are pulling into memory you'd want to select only the fields that you care about. 
2. You do filter and then another filter. You can reduce those to one call to filter. sqlalchemy ends up producing the same query regardless but I think it is a little cleaner to reduce to one.

What do you think of this?
```python
                q = cls._session().query(t, cls).with_entities(
                    t.user_name.label('email'),
                    *cls.__table__.columns,
                ).filter(
                    t.uid == cls.uid,
                    sqlalchemy.sql.expression.or_(cls.status == 'pending', cls.status == 'clarify')
                ).all()
            return [dict(zip(r.keys(), r)) for r in q]
```

Reply:
6a18786a1c0ea80d306b558a3c9d83ef8d1df2d8